addons/hr_holidays/static/src/radio_image_field/radio_image_field.xml

```diff
@@ -12,7 +12,7 @@
             </t>
             <t t-else="">
                 <t t-foreach="items" t-as="item" t-key="item[0]">
-                    <div class="custom-control form-check o_radio_item" aria-atomic="true">
+                    <div class="form-check form-check o_radio_item" aria-atomic="true">
```

Comment:
There are some duplicate left (`form-check form-check` in this case) in the code due to the replacement.